### ISSUE TITLE
Add support for policies

### ISSUE THREAD

**Issue by zambrovski** [chilit-nl/plantuml-event-modeling#2]
Along with command -> concept (aggregate) -> event there is mostly a complement event -> policy -> command. I would like to be able to model this.

*Acceptance:*
- policy should be purple
- the policy lane belongs underneath the concepts (aggregates / entities)

**Comment by marriesvdh** [comment]
Thank you for your contribution! And sorry for the late reply.

Policy is used in Event Storming, but I haven't found it in Event Modeling. So I'm not sure it would fit in this library. But maybe I'm not aware of its usage in Event Modeling. Do you have any references where it is used in Event Modeling?

**Comment by zambrovski** [comment]
Hi.

Policies are actually not part of any method but exist in real world :-). But yes, you are right, it is not part of the official Event Modelling, making the latter not applicable every time I have policies triggering some commands. 

So it is OK to me just to drop the PR... After playing around with it I decided not to use Event Modelling as a method. Was just an idea to share what I tried to build and how you like it.

Cheers,

Simon
